### PR TITLE
 Store theme, too 

### PR DESCRIPTION
**Pull request by Kijewski**
And as it turns out, neither `json` nor `jzon` is needed. :D

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Wait wat. O.o

In my case the backlines were completely removed when I stored without JSON conversion. :o

File: src/app.rs [hunk 1]
```diff
@@ -34,52 +34,37 @@ fn local_storage() -> Option<Storage> {
     }
 }
 
+const THEME_SOURCE_KEY: &str = "theme";
```
Comment (GuillaumeGomez) [inline src/app.rs:37]: It's better to prefix local storage keys with the "application" name. Especially when you test with localhost (which I do a lot with rustdoc). Do you mind putting back the prefix please?

File: src/app.rs [hunk 2]
```diff
@@ -34,52 +34,37 @@ fn local_storage() -> Option<Storage> {
...
-            (struct_source, template_source)
-        } else {
-            (STRUCT_SOURCE, TMPL_SOURCE)
+        let local_storage = local_storage();
+        let local_storage_or = |default: &str, key: &str| -> Rc<str> {
+            let value = local_storage
+                .as_ref()
+                .and_then(|ls| get_data_from_local_storage(ls, key));
+            Rc::from(match value.as_deref().unwrap_or_default() {
+                "" => default,
```
Comment (GuillaumeGomez) [inline src/app.rs:54]: This one is wrong. You might have emptied the whole source code. In this case, you don't want the default to be put back.

File: src/app.rs (hunk 1 above)
Comment (Kijewski) [reply src/app.rs:37]: Will do. :+1:

File: src/app.rs (hunk 2 above)
Comment (Kijewski) [reply src/app.rs:54]: I thought this might be better than putting nothing in there. Will change back to restore the empty data.

**Comment by Kijewski** [comment]
> In my case the backlines were completely removed when I stored without JSON conversion. :o

Oh, I did not test in different browsers. Maybe it should be encapsulated in JSON to be safe. Will change. :/

**Comment by GuillaumeGomez** [comment]
I use firefox if you want to check locally.

**Pull request merged**